specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ContainerApps.json

```diff
@@ -756,6 +766,41 @@
         }
       }
     },
+    "IpSecurityRestrictionRule": {
+      "description": "Rules to restrict incoming IP address.",
+      "type": "object",
+      "required": [
+        "name",
+        "ipAddress",
+        "action"
+      ],
+      "properties": {
+        "name": {
+          "description": "Name for the IP restriction rule.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Describe the IP restriction rules that are being sent to the container-app. This is an optional field.",
+          "type": "string"
+        },
+        "ipAddress": {
+          "description": "CIDR notation to match incoming IP address",
+          "type": "string"
+        },
+        "action": {
+          "description": "Allow or Deny rules to determine for incoming IP",
```

Comment:
We can add a note that only one action type is allowed for complete rule set

Reply:
added

Reply:
Please confirm that this schema is consistent with what other Azure RPs use.